cmake/CMakeLists.txt

```diff
@@ -329,16 +326,17 @@ foreach(PKG_WITH_INCL KSPACE PYTHON VORONOI USER-COLVARS USER-MOLFILE USER-NETCD
 endforeach()
 
 set(CMAKE_TUNE_FLAGS "${CMAKE_TUNE_DEFAULT}" CACHE STRING "Compiler specific optimization or instrumentation")
-separate_arguments(CMAKE_TUNE_FLAGS)
-include(CheckCXXCompilerFlag)
-foreach(_FLAG ${CMAKE_TUNE_FLAGS})
-  check_cxx_compiler_flag("${_FLAG}" COMPILER_SUPPORTS${_FLAG})
-  if(COMPILER_SUPPORTS${_FLAG})
-    target_compile_options(lammps PRIVATE ${_FLAG})
-  else()
-    message(WARNING "${_FLAG} found in CMAKE_TUNE_FLAGS, but not supported by the compiler, skipping")
-  endif()
-endforeach()
+# this is broken for many flags and compilers
+#separate_arguments(CMAKE_TUNE_FLAGS)
+#include(CheckCXXCompilerFlag)
+#foreach(_FLAG ${CMAKE_TUNE_FLAGS})
+#  check_cxx_compiler_flag("${_FLAG}" COMPILER_SUPPORTS${_FLAG})
```

Comment:
I think we need to strip the `=`, `"` etc. out of `_FLAG` with `string(REGEX REPLACE "=\"" "" _FLAG ${_FLAG})` to make this work.